CITATION.cff

```diff
@@ -0,0 +1,43 @@
+cff-version: 1.2.0
+message: "Cite this paper whenever you use Julia"
+authors:
+- family-names: "Bezanson"
+  given-names: "Jeff"
+- family-names: "Edelman"
+  given-names: "Alan"
+- family-names: "Karpinski"
+  given-names: "Stefan"
+- family-names: "Shah"
+  name-particle: "B"
+  given-names: "Viral"
+title: "Julia: A fresh approach to numerical computing"
+version: "v1"
+license: "MIT"
+doi: "10.1137/141000671"
+date-released: 2017-02-07
+url: "https://julialang.org"
+references:
+  - scope: "Cite this paper whenever you use Julia"
+    type: article
+    title: "Julia: A fresh approach to numerical computing"
+    authors:
+      - family-names: "Bezanson"
+        given-names: "Jeff"
+      - family-names: "Edelman"
+        given-names: "Alan"
+      - family-names: "Karpinski"
+        given-names: "Stefan"
+      - family-names: "Shah"
+        name-particle: "B"
```

Comment:
From https://github.com/citation-file-format/citation-file-format/blob/main/README.md#exemplary-uses-2, is "B" a name particle, or is it part of the given names?

@ViralBShah

Reply:
Yikes I don't understand so much about names. It is my middle name (first initial for my father's name) - which I use due to so many Viral Shahs out in the wild. I would say that becomes part of my given name.

Reply:
The example for `name-particle` in the documentation is:

```yaml
- family-names: "Beethoven"
  name-particle: "van"
  given-names: "Wolfgang"
```

Reply:
I looked at the docs - it is not a name-participle in my case. It really should be "Viral B." as my given name. Or if it is all confusing, just "Viral" as the given name.

Reply:
Let's do `Viral B.`